WebSecurityScanner/src/V1beta/WebSecurityScannerClient.php

```diff
@@ -28,11 +28,9 @@
 
 use Google\Cloud\WebSecurityScanner\V1beta\Gapic\WebSecurityScannerGapicClient;
 
-/**
- * {@inheritdoc}
- */
+/** {@inheritdoc} */
```

Comment:
nit: I prefer the expanded docblock. If this is as easy as adding a line to the linter, I think it would be worth doing.